pkg/config/config.go

```diff
@@ -197,7 +197,20 @@ func (vcConfig *VCConfig) validate() error {
 	return nil
 }
 
+func removeEmptyItem(source []string) []string {
+	target := make([]string, 0)
+	for _, value := range source {
+		if len(value) == 0 {
+			continue
+		}
+		target = append(target, value)
+	}
+	return target
+}
+
```

Comment:
target := make([]string, 0, len(source))
	for i, value := range source {
		if len(value) != 0 {
			target[i] = value
		}
	}
	return target

Reply:
It doesn't work. target[i]=value will crash. unless
target := make([]string, len(source))